modules/p2/ting_openlist/include/ting_openlist_analysis.inc

```diff
@@ -0,0 +1,150 @@
+<?php
+
+/**
+ * @file
+ * Functions for analysis of local OpenList Entities
+ */
+
+/**
+ * Get providers.
```

Comment:
Please change the documentation to "provider users" instead of "providers". A provider will usually be module/system which integrates with Ding using the `ding_provider` module.

I know that Drupal users created through the provider system also have the role "provider". This is unfortunate but also hard to change. This is why we need to be precise in our documentation.